app/views/user_mailer/1st_reminder.html.erb

```diff
@@ -0,0 +1,349 @@
+                                </tr>
+                                </tbody></table>
+                            </td>
+                          </tr>
+                          </tbody></table>
+
+                      </td>
+                    </tr>
+                    </tbody>
+                  </table><table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnDividerBlock" style="min-width:100%;">
+                    <tbody class="mcnDividerBlockOuter">
+                    <tr>
+                      <td class="mcnDividerBlockInner" style="min-width:100%; padding:18px;">
+                        <table class="mcnDividerContent" border="0" cellpadding="0" cellspacing="0" width="100%" style="min-width: 100%;border-top: 2px solid #505050;">
+                          <tbody><tr>
+                            <td>
+                              <span></span>
+                            </td>
+                          </tr>
+                          </tbody></table>
+                        <!--
+                                        <td class="mcnDividerBlockInner" style="padding: 18px;">
+                                        <hr class="mcnDividerContent" style="border-bottom-color:none; border-left-color:none; border-right-color:none; border-bottom-width:0; border-left-width:0; border-right-width:0; margin-top:0; margin-right:0; margin-bottom:0; margin-left:0;" />
+                        -->
+                      </td>
+                    </tr>
+                    </tbody>
+                  </table><table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnTextBlock" style="min-width:100%;">
+                    <tbody class="mcnTextBlockOuter">
+                    <tr>
+                      <td valign="top" class="mcnTextBlockInner" style="padding-top:9px;">
+                        <!--[if mso]>
+                        <table align="left" border="0" cellspacing="0" cellpadding="0" width="100%" style="width:100%;">
+                          <tr>
+                        <![endif]-->
+
+                        <!--[if mso]>
+                        <td valign="top" width="600" style="width:600px;">
+                        <![endif]-->
+                        <table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
+                          <tbody><tr>
+
+                            <td valign="top" class="mcnTextContent" style="padding-top:0; padding-right:18px; padding-bottom:9px; padding-left:18px;">
+
+                              © Copyright 2015-2018&nbsp;<a href="https://mumuki.io/home/">Mumuki Project</a><em>.</em><br>
+                              <br>
+                              <%= t :stop_emails? %><br>
+                              <%= link_to t(:cancel_subscription), "https://mumuki.io/user/unsubscribe?id=#{@unsubscribe_code}", target: :_blank %>
```

Comment:
I was just thinking this might conflict with #903, if someone with an immersive main organization or more than one organization uses this link won't they get redirected to central? :thinking:

Should I make this link https://central.mumuki.io/user/unsubscribe?id=#{@unsubscribe_code} in the first place?

Reply:
Right. Also, we should not hardcode the domain.